pymc3/tests/test_distributions.py

```diff
@@ -1371,24 +1389,36 @@ def test_binomial(self):
     @pytest.mark.xfail(
         condition=(SCIPY_VERSION < parse("1.4.0")), reason="betabinom is new in Scipy 1.4.0"
     )
-    def test_beta_binomial(self):
-        self.checkd(
+    def test_beta_binomial_logcdf(self):
+        self.check_logcdf(
             BetaBinomial,
             Nat,
             {"alpha": Rplus, "beta": Rplus, "n": NatSmall},
+            lambda value, alpha, beta, n: sp.betabinom.logcdf(value, a=alpha, b=beta, n=n),
         )
-        self.check_logp(
+
+    @pytest.mark.xfail(
+        condition=(SCIPY_VERSION < parse("1.4.0")), reason="betabinom is new in Scipy 1.4.0"
+    )
+    def test_beta_binomial(self):
+        self.checkd(
             BetaBinomial,
             Nat,
             {"alpha": Rplus, "beta": Rplus, "n": NatSmall},
-            lambda value, alpha, beta, n: sp.betabinom.logpmf(value, a=alpha, b=beta, n=n),
         )
-        self.check_logcdf(
+
+    @pytest.mark.xfail(
+        condition=(SCIPY_VERSION < parse("1.4.0")), reason="betabinom is new in Scipy 1.4.0"
```

Comment:
Would this be better as a `@pytest.mark.skipif`?

Reply:
Possibly yes. Given that there should be no chance for the behavior of `betabinom` to change, given that it won't exist there is no point in monitoring the outcome of these tests, so it may be worth skipping them. Probably by using skipif the testing pipeline would get slightly faster